client_end_user/shared/src/commonMain/kotlin/resources/strings/ar/ArabicEG.kt

```diff
@@ -90,5 +90,7 @@ data class ArabicEG(
     override val orderNow: String = "أطلب دلوقتي",
     override val total: String = "المجموع:",
     override val yourCart: String = "العربة",
-    override val pleaseLogin: String="سجل دخول يا اسطا وشوفك  حسابك"
+    override val pleaseLogin: String="سجل دخول يا اسطا وشوفك  حسابك",
```

Comment:
there is double spaces here
```suggestion
    override val pleaseLogin: String="سجل دخول يا اسطا وشوفك حسابك",
```